README.md

```
@@ -21,57 +52,61 @@ RocketRide is a high-performance data processing engine built on a C++ core with
-3) Connect input lanes and output lanes by type to properly wire your pipeline. Some nodes like agents or LLMs can be invoked as tools for use by a parent node as shown below:
+3. Connect input lanes and output lanes by type to properly wire your pipeline. Some nodes like agents or LLMs can be invoked as tools for use by a parent node as shown below:
 
 <p align="center">
   <img src="./images/agent_pipeline.png" alt="Pipeline canvas example">
 </p>
 
-4) You can run a pipeline from the canvas by pressing the ▶️ button on the source node or from the `Connection Manager` directly.
+4. You can run a pipeline from the canvas by pressing the ▶️ button on the source node or from the `Connection Manager` directly.
 
-5) View all available and running pipelines below the `Connection Manager`. Selecting running pipelines allows for in depth analytics. Trace call trees, token usage, memory consumption, and more to optimize your pipelines before scaling and deploying.
+5. View all available and running pipelines below the `Connection Manager`. Selecting running pipelines allows for in depth analytics. Trace call trees, token usage, memory consumption, and more to optimize your pipelines before scaling and deploying.
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 5; Style: 1/2/3

(MD029, ol-prefix)